### ISSUE TITLE
Validate ENA / SRA IDs in search, support GSE* and GSM* IDs

### ISSUE THREAD

**Issue by pansapiens** [MonashBioinformaticsPlatform/laxy#30]
The ENA / SRA search form should warn users if the IDs aren't of a valid form, or if searched IDs aren't found.

Ideally, users should also be able to search via GEO IDs (GSM* or GSE*).

**Comment by drpowell** [comment]
Agreed - just hit this which returned a 500 error.  Then realised I had the wrong ID type

**Comment by pansapiens** [comment]
Consider using https://github.com/pachterlab/ffq to fetch metadata
